docs/user/ppl/cmd/fillnull.rst

```diff
@@ -0,0 +1,67 @@
+=============
+fillnull
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| The ``fillnull`` command replaces null values for one or more fields.
+
+
+Syntax
+============
+fillnull "with" <expression> <field> ["," <field> ]...
```

Comment:
@normanj-bitquill please review [spark's ppl fillnull](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/ppl-lang/ppl-fillnull-command.md) to make sure we are aligned with the syntax